### DISCUSSION TITLE
S3 backend seems to not retrieve data

### DISCUSSION THREAD

**Discussion by ofiry-tm** [grafana/mimir#4277] [Help and support] [answered]
I use Mimir with s3 backend, 
and I attached Mimir a service-account that is attached to an iam role with these permissions to the bucket:
```
s3:Get*,
s3:List*,
s3:*Object*
```

It seems that the Mimir writes to the Bucket successfully.
But, when i delete the mimir pods, and its persistentVolumes, wait a while, and redeploy mimir,
then i dont see nothing from the old metrics in Grafana.

How do  you explain this ?

**Comment by pracucci** [comment, marked as answer]
Looks like a duplicate of https://github.com/grafana/mimir/issues/4267#issuecomment-1440014973. I will reply there.
